apps/condo/domains/acquiring/schema/SumPaymentsService.test.js

```diff
@@ -0,0 +1,140 @@
+        const {
+            data: {
+                result: {
+                    sum,
+                },
+            },
+        } = await admin.query(SUM_PAYMENTS_MUTATION, { where: where })
+
+        expect(Big(sum).eq(Big(0))).toBeTruthy()
+    })
+    test('support can sum payments', async () => {
+        const { admin, billingReceipts, acquiringContext, organization } = await makePayer(10)
+        const support = await makeClientWithSupportUser()
+        let totalSum = Big(0)
+        for (let i = 0; i < 10; i++){
+            const [payment] = await createTestPayment(admin, organization, billingReceipts[i], acquiringContext)
+            totalSum = totalSum.plus(Big(payment.amount))
+        }
+
+        const where = { organization: { id: organization.id } }
+        const {
+            data: {
+                result: {
+                    sum,
+                },
+            },
+        } = await support.query(SUM_PAYMENTS_MUTATION, { where: where })
+        expect(Big(sum).eq(totalSum)).toBeTruthy()
+    })
+    test('employee with canReadPayments can sum payments', async () => {
+        const { admin, billingReceipts, acquiringContext, organization } = await makePayer(10)
+        const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+            canReadPayments: true,
+        })
+        const employeeClient = await makeClientWithNewRegisteredAndLoggedInUser()
+        await createTestOrganizationEmployee(admin, organization, employeeClient.user, role)
+
+        let totalSum = Big(0)
+        for (let i = 0; i < 10; i++){
+            const [payment] = await createTestPayment(admin, organization, billingReceipts[i], acquiringContext)
+            totalSum = totalSum.plus(Big(payment.amount))
+        }
+        const where = { organization: { id: organization.id } }
+        const {
+            data: {
+                result: {
+                    sum,
+                },
+            },
+        } = await employeeClient.query(SUM_PAYMENTS_MUTATION, { where: where })
+
+
+        expect(Big(sum).eq(totalSum)).toBeTruthy()
+    })
+    test('employee without canReadPayments cant sum payments', async () => {
+        const { admin, billingReceipts, acquiringContext, organization } = await makePayer()
+        const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+            canReadPayments: false,
+        })
+        const employeeClient = await makeClientWithNewRegisteredAndLoggedInUser()
+        await createTestOrganizationEmployee(admin, organization, employeeClient.user, role)
+
+        await createTestPayment(admin, organization, billingReceipts[0], acquiringContext)
+
+        const where = { organization: { id: organization.id } }
+        const {
+            data: { result },
+            errors,
+        } = await employeeClient.query(SUM_PAYMENTS_MUTATION, { where: where })
+
+        expect(result).toBeNull()
+        expect(errors).toHaveLength(1)
```

Comment:
No, this is not the way :-) You should explicitly check for the error signature here